util/test/perf/perfgraph.js

```diff
@@ -167,9 +167,12 @@ var branchInfo = [
                     "releaseDate": "2024-03-21",
                     "branchDate" : "2024-03-15",
                     "revision": -1},
+                    { "release": "2.1.0",
+                      "releaseDate": "2024-06-27",
+                      "branchDate" : "2024-06-24",
+                      "revision": -1}
```

Comment:
this should continue to have a trailing comma, I believe

Reply:
also this entry is over-indented

Reply:
Pushed new changes to resolve these issues.